### PR TITLE
[ZH] Add `Game_mode` and update `FAQ`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

---

The commits are quite messy then... 😓

### CODE REVIEW

File: wiki/FAQ/zh.md [hunk 1]
```diff
@@ -1,365 +1,562 @@
...
+- 银 SS (SSH) = 使用 *[Hidden](/wiki/Gameplay/Game_modifier/Hidden)* 和/或 *[Flashlight](/wiki/Gameplay/Game_modifier/Flashlight)* 模组的正常 SS 等级。
+- 银 S (SH) = 使用 *[Hidden](/wiki/Gameplay/Game_modifier/Hidden)* 和/或 *[Flashlight](/wiki/Gameplay/Game_modifier/Flashlight)* 模组的正常 S 等级。
+
+**关于个人资料页面的*评价*：**
+
+- 当接收等级时，如果游戏内 osu! 账户仍与 Bancho 服务器连接在线，则获得的等级会设置在资料页面的一般->等级下。
+  - **离线时的游玩**，即使 osu! 账户已登录，**也不会被记入**。
+- **资料给出的等级取决于对应游戏模式中上架/达标谱面难度的最高分获得的等级，而非所有分数。**
+  - 如果一张谱面有三 (3) 个上架难度，并且在 osu!catch 中 SS 了所有难度，则 osu!catch 的评价部分会得到三 (3) 个 SS，而不是只得到一 (1) 个。
+  - 然而，如果通过重玩已 SS 的上架难度获得了新的最高分（转盘加分更多，或者是通过游戏模组）但获得了 S 等级，则 SS 总数会减少一 (1)，S 总数会增加一 (1)。
```
Comment (YumeMuzi) [inline wiki/FAQ/zh.md:378]: 不需要加括号来解释吧，原文有的吗？即使原文有这里应该也可以直接用数词替代

File: wiki/FAQ/zh.md [hunk 2]
```diff
@@ -1,365 +1,562 @@
...
+![设置中的键盘小节](/wiki/shared/Options_keyboard.jpg "设置 输入图标，键盘小节")
 
-### 这些 map 太难了，有没有一些简单点的介绍一下？
+#### 游戏结束了？我搞砸了啥？
 
-Pokebis 已经收集了一些适合初学者玩的谱面。您可以在这里[下载](https://osu.ppy.sh/community/forums/topics/5456)。
+*完整解释：[回放](/wiki/Gameplay/Replay)*
 
-同时，您可以[按照难度来对谱面列表进行重新排序](https://osu.ppy.sh/beatmapsets)。最容易的谱面就在列表的最上面。
+在谱面游玩失败后，按 `F1` 可以回放当前失败的游玩过程。当屏幕变暗，中央出现 osu!logo 时回放结束，且不会显示游戏结束屏幕。如果要再次观看回放，按下自定义的默认抑音符/波浪号键 (\`/~)（可以在设置侧边栏中更改；在*按键设置*中的**Quick Retry (hold briefly)**）或按下 `Ctrl`+`R` 一段时间来重看回放。
```
Comment (YumeMuzi) [inline wiki/FAQ/zh.md:448]: 加一个快速重试

File: wiki/Game_mode/osu!/zh.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+- 较早版本的 osu! 常常模拟*应援团*系列游戏中的一些元素，这些元素之后被移除或替换：
+  - 达到新的连击里程碑时的[连击火焰](/wiki/Gameplay/Combo_fire)。
+  - 首个版本的默认皮肤（*osu!default by peppy* 皮肤）使用 v1 皮肤行为，提供了一个几乎与*应援团*系列游戏相同的副本。
+    - 之后被使用 v2 皮肤行为的更好的默认皮肤替代。
+  - v1 和 v2 皮肤行为上最值得注意的不同是转盘。
+    - v1 皮肤行为中的转盘像*应援团*系列游戏中的一样，会自下而上增加转盘计的级别，直到全满为止。
+      - 计时器是转盘内的圆圈，会缩小到转盘中心。当计时圈到达转盘中心时，转盘就会结束。
+    - v2 皮肤行为中的转盘在完成时，会将转盘圈缓慢放大到完全大小并发光。
+      - 计时器是转盘外缘的变色圆圈。当计时圈的颜色完全改变时，转盘就会结束。
+- 在线排行榜启用当天（2007 年 10 月 7 日，2007-10-07），最先上架的三张谱面是：
```
Comment (YumeMuzi) [inline wiki/Game_mode/osu!/zh.md:131]:
```suggestion
- 在线排行榜启用当天（2007 年 10 月 7 日），最先上架的三张谱面是：
```

File: wiki/Game_mode/osu!/zh.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+### 游玩
+
+![osu! 拉烟效果](/wiki/shared/osu_smoke.jpg "拉烟功能的使用")
+
+![osu! 拉烟设置](/wiki/shared/osu_smoke_set.jpg "按键绑定中的拉烟键")
+
+- 取决于难度设定与计时严度，*过早*点击物件会使之强烈振动。
+- 在没有隐藏或可见滑条点的空路径上释放滑条球不会导致断连，也不会得分。滑条判定只会检查是否点击起点处的圆圈，是否收集了滑条点，以及是否成功到达终点处圆圈。
+- 玩 *DS* 时（尤其是高难度关卡），完成转盘是刮坏触摸屏（或者屏幕保护膜）的最佳办法。在 osu! 中，转盘已经得到优化，完成它不会这么麻烦。转盘每分钟可实现的最高转数是 477 转，也是 [Auto](/wiki/Gameplay/Game_modifier/Auto) 模组游玩时的旋转速度，而 [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out) 模组的旋转速度较低，是每分钟 287 转。
```
Comment (YumeMuzi) [inline wiki/Game_mode/osu!/zh.md:113]:
```suggestion
- 玩 *DS* 时（尤其是高难度关卡），完成转盘很容易刮坏触摸屏（或者屏幕保护膜）。osu! 的转盘已经优化，完成它不会这么麻烦。转盘每分钟可实现的最高转数是 477 转，也是 [Auto](/wiki/Gameplay/Game_modifier/Auto) 模组游玩时的旋转速度，而 [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out) 模组的旋转速度较低，是每分钟 287 转。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

File: wiki/FAQ/zh.md (hunk 1 above)
Comment (CloneWith) [reply wiki/FAQ/zh.md:378]: [原文](https://github.com/ppy/osu-wiki/blob/master/wiki/FAQ/en.md?plain=1#L378-L380)

我也不明白为什么用这种方式表达，很奇怪🤣

**Review by YumeMuzi — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**